examples/conversation_addons/tone_detection.js

```diff
@@ -0,0 +1,281 @@
+/**
+ * Copyright 2015 IBM Corp. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+var watson = require('watson-developer-cloud');
+var Promise = require('bluebird');
+
+/**
+ * Thresholds for identifying meaningful tones returned by the Watson Tone Analyzer.  Current values are
+ * based on the recommendations made by the Watson Tone Analyzer at
+ * https://www.ibm.com/watson/developercloud/doc/tone-analyzer/understanding-tone.shtml
+ * These thresholds can be adjusted to client/domain requirements.
+ */
+var PRIMARY_EMOTION_SCORE_THRESHOLD = parseFloat("0.5");
```

Comment:
Although, in this case, it should just be `0.5`, no `parseFloat()` or strings. All numbers in JS are floats, so there's no need to do anything special here.